plugins/reporters/asciidoc/src/main/kotlin/AsciiDocTemplateReporter.kt

```diff
@@ -42,7 +42,7 @@ import org.ossreviewtoolkit.utils.ort.createOrtTempDir
  * [3]: https://github.com/asciidoctor/asciidoctorj
  * [4]: https://docs.asciidoctor.org/asciidoctor/latest/convert/available
  */
-open class AsciiDocTemplateReporter(private val backend: String, override val type: String) : Reporter {
+abstract class AsciiDocTemplateReporter : Reporter {
```

Comment:
> Also, doesn't the prevent the user to use AsciiDocTemplateReporter itself as a reporter, i.e. if one wants to write out AsciiDoc files?

This class was not never offered as a standalone reporter.

> Commit message: Why is it actually beneficial to simplify the constructor here? The code in sub-classes seems to be come rather more than less.

It's been a while, TBH I don't remember why exactly I did this change. But to avoid rebase conflicts I'd rather keep it or revert it separately.

Reply:
> This class was not never offered as a standalone reporter.

Indeed. I was pretty sure it was. But I probably mixed this up with the `PlainTextTemplateReporter` (which could use an Asciidoc plain text template file).

> But to avoid rebase conflicts I'd rather keep it or revert it separately.

Ok.